Lib/fontTools/misc/psCharStrings.py

```diff
@@ -251,23 +251,29 @@ class CharStringCompileError(Exception): pass
 
 class SimpleT2Decompiler(object):
 
-	def __init__(self, localSubrs, globalSubrs):
+	def __init__(self, localSubrs, globalSubrs, private):
 		self.localSubrs = localSubrs
 		self.localBias = calcSubrBias(localSubrs)
 		self.globalSubrs = globalSubrs
 		self.globalBias = calcSubrBias(globalSubrs)
+		self.private = private
```

Comment:
I pass in 'private' at this point because in psCharStrings.py::SimpleT2Decompiler also uses private.getNumRegions().